build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -926,6 +926,11 @@ message Digest {
 
 // ExecutedActionMetadata contains details about a completed execution.
 message ExecutedActionMetadata {
+  // A unique identifier for the execution of an action. This makes it possible
+  // to distinguish different action executions from each other, e.g. to
```

Comment:
I'm curious about the use case for this. My gut feeling is that there are a lot of bad ways that clients could start to depend on something like knowing whether than action was executed or cached.